notebooks/ldb-eda-structure.ipynb

```diff
@@ -0,0 +1,1941 @@
+{
```

Comment:
<p>I believe the scatter plot is already enough for continuous variables. If you'd like to use another analysis, maybe the correlation between the variables would be a good one. You can use it with this code:</p><pre class="ql-syntax" spellcheck="false">s1&nbsp;=&nbsp;pd.Series(test['SalePrice'])
s2&nbsp;=&nbsp;pd.Series(test['MasVnrArea'])
s1.corr(s2)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>